app/src/pages/prompt/PromptLayout.tsx

```diff
@@ -45,9 +47,47 @@ const mainCSS = css`
   }
 `;
 
-export function PromptPage() {
-  const loaderData = useLoaderData() as promptLoaderQuery$data;
+/**
+ * Given a tab index and a promptId, return the URL for that tab.
+ */
+function makePromptUrl({
+  index,
+  promptId,
+}: {
+  index: number;
+  promptId: string;
+}) {
+  if (index === 1) {
+    return `/prompts/${promptId}/versions`;
+  } else {
+    return `/prompts/${promptId}`;
+  }
+}
```

Comment:
nit: I think you can just place this inline on the onChange handler since it's so specific to the tab indexes. It might just be easier to read it there.